src/gui/foldercreationdialog.cpp

```diff
@@ -37,7 +40,9 @@ FolderCreationDialog::FolderCreationDialog(const QString &destination, QWidget *
 
     const QString suggestedFolderNamePrefix = QObject::tr("New folder");
 
-    const auto newFolderFullPath = _destination + "/" + suggestedFolderNamePrefix;
+    qInfo(lcFolderCreationDialog) << "Before setting newFolderNameEdit";
```

Comment:
Do we really want to introduce logs just for that? I mean they don't bring much value.

Reply:
@er-vin @FlexW Now, as we have an explanation, I guess they can be removed.